code/game/striketeams/Judge

```diff
@@ -0,0 +1,162 @@
+//Judge
+
+var/list/response_team_members = list()
+
+/datum/striketeam/Judge
+	striketeam_name = Judge
+	faction_name = "Nanotrasen"
+	mission = "preside over this case and come up with an equitable decision."
+	team_size = 1
+	min_size_for_leader = 0//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	spawns_name = "Judge"
+	can_customize = TRUE
+	logo = "Judge-logo"
+	outfit_datum = /datum/outfit/striketeam/judge
+
+/datum/striketeam/Judge/failure()
+	command_alert(/datum/command_alert/judge_fail)
+
+/datum/striketeam/Judge/extras()
+	command_alert(/datum/command_alert/ert_success)
+
+/datum/striketeam/Judge/greet_commando(var/mob/living/carbon/human/H)
+	if(H.key == leader_key)
+		to_chat(H, "<span class='danger'>You are [H.real_name], the Judge. You answer only to Central Command and are expected to follow Space Law. Assist your assigned station to the best of your abilities.</span>")
+	else
+		to_chat(H, "<span class='danger'>You are [H.real_name], Paralegal. You answer only to Central Command and are expected to follow Space Law. Assist your assigned station to the best of your abilities.</span>")
+	for (var/role in H.mind.antag_roles)
+		var/datum/role/R = H.mind.antag_roles[role]
+		R.AnnounceObjectives()
+
+/datum/striketeam/Judge/create_commando(obj/spawn_location, leader_selected = 0, key = "")
```

Comment:
you're completely recycling the ERT spawn here, meaning that judges will come to the station after fully walking through the candyshop of ERT gear... delay trial by another 10 minutes as he chooses his favorite selection of guns, battering ram, tools, autoinjectors, and SCP

Reply:
Yes